tests/hmac.cc

```diff
@@ -83,6 +116,10 @@ TEST_P(HmacKAT, TryKAT)
   if (test_case.full_size == 20) {
     Hacl_HMAC_legacy_compute_sha1(
       tag.data(), key, test_case.key.size(), msg, test_case.msg.size());
+  } else if (test_case.full_size == 28) {
+    std::cout << "Skipping \"full_size=" << test_case.full_size << "\""
```

Comment:
`full_size` is a little confusing here. Maybe something like digest size?

Reply:
Yeah. I kept the name because it comes from https://github.com/cryspen/hacl-packages/pull/140/files#diff-0b41758c3ed2cbd3ccf1a0db71918285c1d4d988c84e76f25a8ff59de890a0aaR31. I could change it to "digest_size" everywhere.

Reply:
I'll leave it as is.